daemon/daemon_unix.go

```diff
@@ -779,6 +779,10 @@ func initBridgeDriver(controller libnetwork.NetworkController, config *Config) e
 	return nil
 }
 
+func (daemon *Daemon) getLayerInit() func(string) error {
+	return daemon.setupInitLayer
```

Comment:
Hmm.., I had not noticed daemon_solaris.go. Looks like we should put it in there as well. Looks like CI does not builds for solaris? Otherwise build should have failed.

Anyway, I will update patch to take care of solaris as well.